package/batocera/emulators/dolphin-emu/dolphin-emu.mk

```diff
@@ -16,4 +21,11 @@ DOLPHIN_EMU_CONF_OPTS += -DBUILD_SHARED_LIBS=OFF
 DOLPHIN_EMU_CONF_OPTS += -DENABLE_EGL=OFF
 DOLPHIN_EMU_CONF_OPTS += -DDISTRIBUTOR='batocera.linux'
 
+ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_ODROIDN2),y)
```

Comment:
ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_ODROIDN2)$(BR2_PACKAGE_BATOCERA_TARGET_VIM3),y)

you can do this